setup.py

```diff
@@ -15,7 +15,8 @@ def readlocal(fname):
 DESCRIPTION = "N-D labeled arrays in Python"
 LONG_DESCRIPTION = readlocal("README.rst")
 SETUP_REQUIRES = []
-INSTALL_REQUIRES = ['numpy >= 1.13', 'pandas >= 0.13.1']
+# excludes numpy 1.16.* because it is incompatible with pytables 3.4
```

Comment:
AFAICT, it dates back much further than 3.4. I would write something like "pytables < 3.5 (which is not released yet)"